Makefile

```diff
@@ -25,6 +25,7 @@ PYTHON             = python
 COMPILER_MODE      = FAST
 PRECISION          = DOUBLE
 OPENMP             = 1
+SNAPPRECISION	   = DOUBLE
```

Comment:
Can we use another name please, at least in the Makefile. I think that this will cause confusion because people may think it affects the .su format, when it's affecting python. Can we go for PY_PRECISION or PYSNAP_PRECISION or something?